frontend/packages/core/src/Feedback/note.tsx

```diff
@@ -24,10 +24,10 @@ export interface NoteConfig extends NoteProps {
 
 interface NotePanelProps {
   direction?: "row" | "column";
-  notes: NoteConfig[];
+  notes?: NoteConfig[];
```

Comment:
should something like `children?: React.ReactElement` be part of the props here?

Reply:
We get this for free with the functional component type interface